### DISCUSSION TITLE
The SageTV program guide for my 3 PBS stations are transposed/do not match what is playing/recording.

### DISCUSSION THREAD

**Discussion by ghost** [OpenSageTV/Community#45] [Docker]
The SageTV program guide for my 3 PBS stations are transposed/do not match what is playing/recording. 
Both the channel numbers and the program playing match with my cable box.
The SageTV SD program guide are mis-matched.
Back in 2022, with my Win10 version, Schedules Direct said the problem is not on their end.

Spectrum Cable box channel 1276, UNCNC, & the program it’s playing, match what SageTV is playing, but the SD guide listing is for channel 1277 UNC-EX.
Cable channel 1277, UNC-EX & its program playing match SageTV playing program,  but the SD guide listing is for 1276 UNCNC.
The guides need to be swapped but, if possible, keep the channel number the same (to match cable box)

My Main PBS station UNCHD 1221 airing & guide is correct and matches the Cable box.
Its duplicate channel #13 has program data from totally different PBS Explorer channel. 
Can the SD Guide data from 1221 be copied to channel 13 for ease of use?

Makes it frustrating to schedule, via the guide, Antiques Roadshow and ending up with Sesame Street. 

My old Sage drive crashed, so I’ve put Sage Docker on my Unraid, using placeshifter, and am still transitioning my stuff into it.  Most of the time Unraid throws me for a loop & I don’t know where to look to resolve the issue.  Am using cable card, HDHR with 3 tuners, and need to reinstall the HD-PVR and maybe GetChannels.

 If you can help, THANKS.

**Comment by PhilipCRoberts** [comment]
I had a problem with my schedule recently as well.  I contacted Schedules Direct, and they said it was upstream from them.  My schedule wasn't incorrect though, it was just plain missing.  It corrected itself in a couple hours.
